fdbserver/ApplyMetadataMutation.cpp

```diff
@@ -134,6 +134,7 @@ class ApplyMetadataMutationsImpl {
 	std::unordered_map<UID, StorageServerInterface>* tssMapping = nullptr;
 
 	std::map<TenantName, TenantMapEntry>* tenantMap = nullptr;
+	std::map<int64_t, TenantName>* tenantIdIndex = nullptr;
```

Comment:
Do we need this map to be ordered? If not, can we make it an unordered_map?

Reply:
Thanks for pointing out, I first thought of doing a range-scan over the map for deletion, but then I piggybacked on `tenantMap` start-end iterator for deletes, so it is more efficient to use `unordered_map` here.